synapse/rest/client/room_batch.py

```diff
@@ -216,6 +220,10 @@ async def on_POST(
         prev_state_ids = list(prev_state_map.values())
         auth_event_ids = prev_state_ids
 
+        # Make the state events float off on their own
+
+        prev_event_id_for_state_chain = generate_fake_prev_event_id()
```

Comment:
We still have the state events float off on their own, but now their in a floating chain instead of individual.